tests/unit/test_zowe_core.py

```diff
@@ -609,10 +609,13 @@ def test_profile_loading_with_invalid_schema(self, get_pass_func):
         """
         # Setup - copy profile to fake filesystem created by pyfakefs
         with self.assertRaises(ValidationError):
-            custom_file_path = os.path.join(self.custom_dir, "invalid.zowe.config.json")
-            shutil.copy(self.original_invalid_file_path, custom_file_path)
-            shutil.copy(self.original_invalid_schema_file_path, self.custom_dir)
+            custom_file_path = os.path.join(self.custom_dir, "zowe.config.json")
             os.chdir(self.custom_dir)
+            with open(self.original_file_path, 'r') as f:
+                original_config = commentjson.load(f)
+            original_config["$schema"] = "fixtures/invalid.zowe.schema.json"
+            with open(os.path.join(self.custom_dir, "invalid.zowe.config.json"), 'w') as f:
```

Comment:
I can't remove the files like `invalid.zowe.config.json` or `invalidUri.zowe.config.json` because other tests were failing

Reply:
The other tests are probably failing because the `setUp` stage still expects the "invalid" files to exist:
https://github.com/zowe/zowe-client-python-sdk/blob/66ceb940535a06afa4ad8e34e43043dafaf39f5c/tests/unit/test_zowe_core.py#L201-L204

I think this can be fixed by moving the `add_real_file` calls for `invalid.zowe.config.json` inside the specific tests where the files are created.